app/views/users/show.html.erb

```diff
@@ -1 +1,9 @@
-<h1>You are in show page of users_view</h1>
+<%= render @user %>
+<p>
+    <strong class="bio">Details: </strong>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Repellat odio harum reiciendis magnam, architecto voluptates ex doloremque ea optio quibusdam rem repudiandae est, fuga mollitia at sequi repellendus illum vitae. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione libero consequuntur nobis natus quia suscipit magni, dolores molestiae iusto reprehenderit repellat saepe itaque, quam sequi voluptate fugit distinctio minima aspernatur?
+</p>
+<%= render @posts %>
```

Comment:
* As shown in the wireframe provided you are to render three of the users most recent posts you can write something like this ``<% @user.recent_post.each do |post| %>``
![image](https://github.com/bilalrajput09/Blog_app_rails/assets/52497246/34878a4f-f180-4da0-af75-c36285c0f167)

Reply:
Fixed